Composer/packages/extensions/obiformeditor/src/schema/uischema.ts

```diff
@@ -103,6 +98,12 @@ export const uiSchema: { [key in SDKTypes]?: UiSchema } = {
   [SDKTypes.IfCondition]: {
     'ui:hidden': ['actions', 'elseActions', ...globalHidden],
   },
+  [SDKTypes.SetProperties]: {
+    assignments: {
+      'ui:field': 'CustomObjectArrayField',
```

Comment:
After looking at the schema, it's not clear to me why this new field is needed after #1617

Reply:
I don't think we need a new field either. Take a look at the `uiSchema` for `SDKTypes.SwitchCondition`. We should be able to do something similar for `SDKTypes.SetProperties`, `SDKTypes.QnAMakerDialog`, and `SDKTypes.RepeatDialog`.

```javascript
[SDKTypes.SetProperties]: {
  cases: {
    'ui:options': {
      object: true,
    },
    items: {
      'ui:options': {
        hideDescription: true,
        inline: true,
      },
      property: {
        'ui:options': {
          hideLabel: true,
          transparentBorder: true,
        },
      },      
      value: {
        'ui:options': {
          hideLabel: true,
          transparentBorder: true,
        },
      },
    },
  },
  'ui:hidden': ['default', ...globalHidden],
},
```

Reply:
Use `ui:options` to replace `customObjectArrayField`